src/assets/styles/base/container.css

```diff
@@ -1,20 +1,23 @@
 .container {
 	min-width: 320px;
+	margin: 0 auto;
+	padding: 0 24px;
 }
 
-@media screen and (min-width: 320px) {
-	.container {
-		width: 100%;
-		margin: 0 auto;
-		padding: 0 24px;
-	}
-}
 
 @media screen and (min-width: 1024px) {
+
 	.container {
-		width: 100%;
-		margin: 0 auto;
 		padding: 0 150px;
 	}
+
 }
 
+@media screen and (min-width: 1440px) {
+
+	.container {
+		max-width: 1140px;
+		padding: 0;
+	}
+
```

Comment:
- remove empty line
- should be define width for breakpoint 1440px

Reply:
I made changes as required. 
- Grateful -